metadata-models/src/main/resources/entity-registry.yml

```diff
@@ -174,4 +174,39 @@ entities:
     keyAspect: dataPlatformInstanceKey
     aspects:
       - status
+  - name: mlModel
+    category: internal
```

Comment:
qq: are these internal? internal usually used for things like policies and secrets that the outside world shouldn't generally look at

Reply:
Oh ill fix that